pkg/migrations/mls/20250604154813_add_refresh_cursor_to_welcomes.down.sql

```diff
@@ -0,0 +1,5 @@
+SET
+    statement_timeout = 0;
+
```

Comment:
Bun migrations need to have a `bun:split` comment between statements. Look at the other migrations for reference.

Reply:
Got it